Dockerfile

```diff
@@ -3,6 +3,11 @@ ENV LANG C.UTF-8
 
 ARG BUNDLE_GITHUB__COM
 
+# The artsy/ruby:2.7.3-node-14-chrome image has outdated trust keys for Google
+# so, we need to add the updated ones manually
+# https://www.google.com/linuxrepositories/
+RUN wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | tee /etc/apt/trusted.gpg.d/google.asc >/dev/null
```

Comment:
Why not just update the [docker image](https://github.com/artsy/docker-images/tree/main/ruby/2.7.3-node-14-chrome)? In short, I was having trouble getting it to build due to some ruby dependency issues, which could cascade into this image. I am also starting to become skeptical that we should be maintaining so many custom docker images and would like to come back to that (have added to the [platform practice topics](https://www.notion.so/artsy/Platform-Practice-1b558d0627444c9c9bf7ed7583767ca2)). This version of ruby is also past its [EOL date](https://www.ruby-lang.org/en/downloads/branches/).

However, let me know if you feel strongly otherwise.